datastore/storage/nats_test.go

```diff
@@ -0,0 +1,34 @@
+package storage
+
+import (
+	"os"
+	"testing"
+
+	"github.com/nats-io/nats-server/v2/server"
+	natsserver "github.com/nats-io/nats-server/v2/test"
+)
+
+func RunDefaultServer() *server.Server {
+	opts := natsserver.DefaultTestOptions
+	opts.Port = 7030
+	opts.Cluster.Name = "testing"
+	return natsserver.RunServer(&opts)
+}
+
+func TestNATSSet(t *testing.T) {
+	s := RunDefaultServer()
```

Comment:
Do you need to shutdown the server after the test?

Reply:
Had the same question but there's no Close nor Stop methods whatsoever, so I guess not.